docs/rfc/000-Payload-v37.md

```diff
@@ -24,8 +24,11 @@ type PayloadAlpha37 = [
   // we can save more space
   authorNetwork: UTF8String | SocialNetworkEnum,
   authorID: UTF8String,
-  authorPublicKey: ArrayBuffer,
-  // authorPublicKeyCurve: ???,
+  authorPublicKey: ArrayBuffer,                 // format: [PublicKey, SupportedCurvesEnum]
+  // array of base58 strings of newly generated ephemeral public keys
+  // indexed by SupportedCurves
+  authorEphemeralPublicKeyList: string[],       // length = Object.keys(SupportedCurves).length 
+  // authorPublicKeyCurve no longer necessary
```

Comment:
```suggestion
  authorPublicKey: ArrayBuffer,
  authorPublicKeyCurve: SupportedCurvesEnum,
  authorEphemeralPublicKeyList: ArrayBuffer[],
```

Reply:
why can't we encode the public key to `publicKey-curve`, e.g. `0x0Bd793EA8334A77b2BfD604DbAEdca11eA094306-secp256k1` or `pub17lgvvkwxlruqpfr6hzcy0wtg9s4nn7lpkwn0tz0kct9uzr624deqqdhsj8-ed25519`

Reply:
because no need to do this. we have a field for this so it can be parsed easier

Reply:
But what I want to do is to support a more general encoding of a public key/address in a single string to be used somewhere else, e.g. in Tessercube, Twidere or any other supported systems.

Reply:
is it a standard to represent keys like this? And another thing to mention, instead of storing a string, I tend to store a binary (if is _is_ a binary)

Reply:
Yes it can be found elsewhere like https://handbook.scuttlebutt.nz/concepts/identity. The only difference here is that they were using `.` as the separator but I think `-` is more URL-friendly. Binary sounds okay but since you are using msgpack, what's the difference then?

Reply:
You see, `0x0Bd793EA8334A77b2BfD604DbAEdca11eA094306` is representing an binary data. Converting it into string costs more spaces.